src/BinSkim.Driver/MultithreadedAnalyzeCommand.cs

```diff
@@ -224,7 +224,7 @@ public override int Run(AnalyzeOptions analyzeOptions)
             // interesting (it will always be set).
 
             return analyzeOptions.RichReturnCode == true
-                ? (int)((uint)result & ~(uint)RuntimeConditions.RuleNotApplicableToTarget)
+                ? (int)((uint)result & ~(long)RuntimeConditions.RuleNotApplicableToTarget)
```

Comment:
>long [](http://example.com/codeflow?start=41&length=4)

this is needed by other change.

Reply:
Looking at what the code is doing, we look at the bitvector of `RuleNotApplicableToTarget`, negate it (why?), and conjuct with `result`. Why not cast `result` to `long` as well? Otherwise the bit vectors are of unequal length. It's also strange that we finally cast it down to 32-bit signed `int`. The additional bits in `RuleNotApplicableToTarget` then make no difference?

@michaelcfanning we might need your feedback here.